### DISCUSSION TITLE
Potential social interaction concepts

### DISCUSSION THREAD

**Discussion by gfodor** [mozilla/hubs#680] [General Questions & Discussion] [answered]
- Add 'facial reactjis' which allow a user to send a directed facial expression which implies a certain social cue (eg, "I noticed you" or "I am thinking the same thing")

- Introduce a hand proxy for body control, where the non-dominant hand becomes a tool the user can use to orient their body or spatial posturing

**Comment by pattersonbl2** [comment, marked as answer]
Create feature request : https://github.com/mozilla/hubs/issues/4488
Marking as answered.
